### PR TITLE
UI block update for articles opco home page

### LINKED ISSUE #548: [Product Opco Home] UI block update for related articles

### PR DESCRIPTION
**Pull request by sharanagoudapatil**
Please always provide the [GitHub issue(s)](../issues) your PR is for, as well as test URLs where your change can be observed (before and after):

Fix #548 

Test URLs:
- Before: https://main--danaher-ls-aem--hlxsites.hlx.page/
- After: https://548-ui-block-for-articles-list--danaher-ls-aem--hlxsites.hlx.page/us/en/products/brands/opco-home

### CODE REVIEW

**Review by rgravitvl — APPROVED** [review]

File: blocks/articles-list/articles-list.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+  articles = articles.sort((item1, item2) => item2.publishDate - item1.publishDate).slice(0, 3);
+  const cardList = ul({
+    class:
+          'container grid max-w-7xl w-full mx-auto gap-6 grid-cols-1 sm:grid-cols-1 lg:grid-cols-3 px-3 sm:px-0 justify-items-center mt-3 mb-3',
+  });
+  articles.forEach((article, index) => {
+    cardList.appendChild(createCard(article, index === 0));
+  });
+  block.textContent = '';
+  block.classList.add('pb-12');
```
Comment (mhaack) [inline blocks/articles-list/articles-list.js:25]: For other places I think we used `pb-10`

**Review by mhaack — APPROVED** [review]
The "see all >" is missing next to the heading.

**Review by davenichols-DHLS — APPROVED** [review]
Looks brilliant, thanks Sharan.  Please merge.

**Comment by sharanagoudapatil** [comment]
@mhaack & @davenichols-DHLS - Added 'See all' link as requested by you. Thanks.

**Pull request merged**